### PR TITLE
fix for back button being hidden in skill version page

### LINKED ISSUE #2949: Back button hidden behind susi chat bubble in skill version page
**Actual Behaviour**
The back button is hidden behind the Susi chat bubble in the skill version history page and is not usable

**Expected Behaviour**
The button should be properly visible.

**Steps to reproduce it**
go to any skill
then to skill versions

**LogCat for the issue**

**Screenshots of the issue**
Screen Shot 2019-10-14 at 12 24 59 AM

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by Tanish0019**
Fixes #2949 

Changes:
/src/components/cms/SkillVersion/SkillVersion.js: Placed the back button at top right corner.

Demo Link: https://pr-2950-fossasia-susi-web-chat.surge.sh

Screenshots of the change:
Screen Shot 2019-10-14 at 12 27 13 AM

### CODE REVIEW

File: src/components/cms/SkillVersion/SkillVersion.js [hunk 1]
```diff
@@ -15,11 +15,11 @@ import TableBody from '@material-ui/core/TableBody';
 import TableCell from '@material-ui/core/TableCell';
 import TableHead from '@material-ui/core/TableHead';
 import TableRow from '@material-ui/core/TableRow';
-import _Button from '@material-ui/core/Button';
+import Button from '@material-ui/core/Button';
```
Comment (samagragupta) [inline src/components/cms/SkillVersion/SkillVersion.js]: Do not change this.

Comment (Tanish0019) [reply src/components/cms/SkillVersion/SkillVersion.js]: There are 2 types of button one with style and one without. I have written the styled component for the styled button below. What other approach do you suggest?

**Comment by Tanish0019** [comment]
Screen Shot 2019-11-04 at 1 54 11 AM
@mariobehling Please review :)

File: src/components/cms/SkillVersion/SkillVersion.js [hunk 2]
```diff
@@ -190,41 +182,58 @@ class SkillVersion extends Component {
...
+                >
+                  <Button variant="contained" color="primary" size="small">
+                    Back
+                  </Button>
+                </Link>
+              </div>
+            </div>
             <div style={{ margin: '0.625rem' }}>{commitHistoryTable}</div>
-            <ActionButtonDiv>
+            {/* <div>
```
Comment (akshatnitd) [inline src/components/cms/SkillVersion/SkillVersion.js]: Can you remove this comment? Rest looks fine.

**Pull request merged**